views/engagement.html

```diff
@@ -1,17 +1,19 @@
 {{<partials-layout}}
-  {{$menu}}
-    <main id="content" class="main" role="main">
-      <ul class="nav-menu">
-        <li><a href="/">What we do</a></li>
-        <li><a href="/hosting platform">Hosting Platform</a></li>
-        <li><a href="/teams">Teams</a></li>
-        <li>Engagement</li>
-        <li><a href="/best-practices">Best practices</a></li>
-        <li><a href="/roadmap">Road map</a></li>
-        <li><a href="/resources">Resources</a></li>
-      </ul>
-  {{/menu}}
-  {{$content}}
+{{$menu}}
+<div class="navbar">
```

Comment:
Yeah it's totally mental :face_with_head_bandage:  .  Every menu is the same except when it refers to that particular page.  I'll try to see how to do this better